Assets/Engine/Schema/enginedependency.xmlschema

```diff
@@ -1,38 +0,0 @@
-<ObjectStream version="3">
```

Comment:
Why are you deleting this xml schema file? Isn't the _dependencies.xml feature still available? This should probably be kept.

Reply:
good catch!